App.js

```diff
@@ -31,30 +31,38 @@ import LoginPage from './components/auth/loginPage';
 import SignupPage from './components/auth/signupPage';
 
 import fakeUser from './data/profileData.js';
-import data from './data/data';
 const Tab = createBottomTabNavigator();
 
 export default function App() {
   // Determine user has light/dark theme on phone
   let phoneTheme = true;
   const colorScheme = Appearance.getColorScheme();
-  if (colorScheme === 'dark') {
+  if (colorScheme === 'light') {
     phoneTheme = false;
 }
 
   // Add State that will be shared globally here
-  const [name, setName] = useState(fakeUser.first_name);
+  const [userEmail, setUserEmail] = useState('');
+  const [userId, setUserId] = useState('');
+  const [fetchdata, setFetchData] = useState('');
+  const [friendsList, setFriendsList] = useState('');
+  const [token, setToken] = useState('');
```

Comment:
Is this the authentication token? If so, we may want to research whether or not it's safe to store it in state.